### PR TITLE
Sentinel nerf(no s) tackle chance

### PR DESCRIPTION
**Pull request by irakligabunia**
# About the pull request

Makes it so sentinel can't 100% cap you all of the time

# Explain why it's good for the game

A T1 caste is able to hit you once and then cap you. In my opinion, that is an extremely strong ability for such a stun heavy T1.
This change makes it not guaranteed, and adds risk to it for Sentinels.
Sincerely; 

# Testing Photographs and Procedure
nubmerchange

</details>

# Changelog
:cl: 
balance: sets sentinel maximum tackle to 6 to alleviate one ability capping
/:cl:

### CODE REVIEW

**Comment by NexusStorm1** [comment]
This nerf will make sentinel weaker than it was pre-buff, since you now potentially need 2 extra tackles even though the duration buff on the paralyzing slash only gives 1 extra tackle, along with the unreliable RNG forcing you to play safer. Also, have you tested when the sentinel gets bad RNG (6 tackles), will they still be able to drag and knock down queen screech stunned marines (doubt it), and can they keep a marine perma-tackled (unsure)? There are pretty big implications for the caste if either of these two are no longer possible.

**Comment by irakligabunia** [comment]
> This nerf will make sentinel weaker than it was pre-buff, since you now potentially need 2 extra tackles even though the duration buff on the paralyzing slash only gives 1 extra tackle, along with the unreliable RNG forcing you to play safer. Also, have you tested when the sentinel gets bad RNG (6 tackles), will they still be able to drag and knock down queen screech stunned marines (doubt it), and can they keep a marine perma-tackled? Also, will the current tackle stun be long enough for the sentinel to start up the devour action on a tackled-down marine AND get six additional tackles? There are pretty big implications for the caste if either of these are no longer possible.

yes you can still perma tackle

**Review by irakligabunia — COMMENTED** [review]
i forgot to addd this

**Comment by NexusStorm1** [comment]
> > This nerf will make sentinel weaker than it was pre-buff, since you now potentially need 2 extra tackles even though the duration buff on the paralyzing slash only gives 1 extra tackle, along with the unreliable RNG forcing you to play safer. Also, have you tested when the sentinel gets bad RNG (6 tackles), will they still be able to drag and knock down queen screech stunned marines (doubt it), and can they keep a marine perma-tackled? Also, will the current tackle stun be long enough for the sentinel to start up the devour action on a tackled-down marine AND get six additional tackles? There are pretty big implications for the caste if either of these are no longer possible.
> 
> yes you can still perma tackle

What about the other 2 scenarios I mentioned? And would an unlucky sentinel still be able to tackle down a marine that they've regurgitated during the regurgitation stun? I'm just dropping some scenarios for you to consider, since I'm afraid that there could be many unintended gameplay side effects from such a change.

**Comment by TheManWithNoHands** [comment]
Am i blind?
I can only see a change to max in the code changes, and non to the min you mentioned in the changelog.

**Comment by deathrobotpunch** [comment]
> Am i blind? I can only see a change to max in the code changes, and non to the min you mentioned in the changelog.

there was no change to min at all, min was already set to 4

File: code/modules/mob/living/carbon/xenomorph/castes/Sentinel.dm [hunk 1]
```diff
@@ -19,10 +19,10 @@
 	acid_level = 1
 
 	tackle_min = 4
-	tackle_max = 4
+	tackle_max = 6
 	tackle_chance = 50
 	tacklestrength_min = 4
-	tacklestrength_max = 4
+	tacklestrength_max = 6
```
Comment (Drulikar) [inline code/modules/mob/living/carbon/xenomorph/castes/Sentinel.dm:25]: Are you sure about the `tacklestrength_max` change? Its `rand(attacking_xeno.tacklestrength_min, attacking_xeno.tacklestrength_max)` for determining the knockdown duration for a successful tackle. Just asking if you were intentionally increasing how long they knock you over.

**Comment by TheManWithNoHands** [comment]
> there was no change to min at all, min was already set to 4

Ahh... @irakligabunia fix your changelog.

File: code/modules/mob/living/carbon/xenomorph/castes/Sentinel.dm (hunk 1 above)
Comment (Drulikar) [reply code/modules/mob/living/carbon/xenomorph/castes/Sentinel.dm:25]: If intentional, the CL needs to reflect this.

**Comment by Drulikar** [comment]
Failure to comply to requested changes. Please learn what `tacklestrength_max` does.

**Pull request closed without merging**